app/pages/SendReceive/BurnTokens.view/BurnTokens.view.tsx

```diff
@@ -35,6 +42,10 @@ const BurnTokens: FC = () => {
   const fee = Number(fees[token.id]);
   const balance = Number(selectedAccount.balanceStatus.balancePerToken[token.id].unspentPmob);
   const submitEnabled = amount > 0 && !amountError && memo.length && !memoError;
+  const { viewOnly } = selectedAccount.account;
+  const buttonText = viewOnly ? 'Save Unsigned Burn Transaction' : 'Build Burn Transaction';
+  const localBlockHeightBigInt = Number(selectedAccount.balanceStatus.localBlockHeight ?? 0);
+  const offlineTombstone = `${localBlockHeightBigInt + MAX_TOMBSTONE_BLOCK}`;
```

Comment:
the default value here on minting tombstone blocks would be `localBlockHeightBigInt + MAX_TOMBSTONE_BLOCK - 1`. For safety, lets switch to that implementation here. Burning may be different, I'm not sure.